ghost/core/core/server/services/mega/post-email-serializer.js

```diff
@@ -336,8 +336,10 @@ const PostEmailSerializer = {
 
         // Also replace the links in the HTML version
         if (labs.isSet('emailClicks')) {
-            if ((!options.isBrowserPreview && !options.isTestEmail) || process.env.NODE_ENV === 'development') {
-                content.html = await linkReplacement.service.replaceLinks(content.html, newsletter, postModel);
+            if (settingsCache.get('email_track_clicks')) {
```

Comment:
I think that we want this check in the link-tracking service rather than here, as we still want to be able to update links for sent emails

Reply:
I agree, and added it in here on purpose until we moved it to a better suitable place in the future 😊

Reply:
I moved it to a more permanent location in the `LinkReplacementService`. We can still decide to move it to the `LinkClickTrackingService`, but not super sure about that (and that is something we can easily change later)